lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -6119,29 +5924,48 @@ class PullbackEmitter final : public SILInstructionVisitor<PullbackEmitter> {
     return lastLocalAlloc->getDefiningInstruction()->getIterator();
   }
 
-  SILValue &getAdjointBuffer(SILBasicBlock *origBB, SILValue originalBuffer) {
-    assert(originalBuffer->getType().isAddress());
-    assert(originalBuffer->getFunction() == &getOriginal());
-    auto insertion = bufferMap.try_emplace({origBB, originalBuffer},
+  SILValue makeLocalAllocation(SILLocation loc, SILType type) {
```

Comment:
I couldn't find a single caller of this method. Dead code?

Reply:
Thanks - removed in https://github.com/apple/swift/pull/26709/commits/bc2657784cb232a2f85921b8446883762a079f18.